firefox-ios/Shared/Strings.swift

```diff
@@ -4566,6 +4592,11 @@ extension String {
                     tableName: "MainMenu",
                     value: "Page Zoom (%@)",
                     comment: "On the main menu, in the tools submenu, the title for the menu component that indicates the current page zoom level. %@ is the current zoom level percentage. ")
+                public static let PageZoomV2 = MZLocalizedString(
+                    key: "MainMenu.Submenus.Tools.PageZoomV2.Title.v141",
```

Comment:
nit: should be without `Submenus.Tools`? now that is on the top level
```suggestion
                    key: "MainMenu.Submenus.Tools.PageZoomV2.Title.v141",
```
```suggestion
                    key: "MainMenu.PageZoom.Title.v141",
```